docs/source/models/cause.rst

```diff
@@ -16,6 +16,62 @@ What is a cause?
 The structure of a cause model
 ------------------------------
 
+Cause model impact 
+-------------
+
+1. **What is model impact**
+ Model impact is about how causes affect mortality/disability
+2. **Mortality**
```

Comment:
@yaqi7 The content here is good because it operationally describes how the data gets used in the simulation, but we also want to write some information about what the actual quantities that get used are.  

In particular, you want to think about what the cause specific mortality rate is and what the excess mortality rate is.  I think it's probably useful to have a section on case fatality rates as well.  The format ali has is useful to frame this.  E.g.

The cause specific mortality rate is a measure of ....

For example the CSMR due to X is Y.  In a population of 10K people over 1 year, this means....

In the simulation we use the CSMR to ...

In the simulation the CSMR modifies and indivuals mortality rate by...